imports/plugins/core/shipping/client/templates/checkout/shipping.js

```diff
@@ -65,7 +66,8 @@ Template.coreCheckoutShipping.onCreated(function () {
   const shippingOpts = {
     provides: "settings",
     name: "settings/shipping",
-    template: "shippingSettings"
+    template: "shippingSettings",
+    shopId: Reaction.getPrimaryShopId()
```

Comment:
Could this potentially permit a non-marketplace-owner to adjust the shipping settings for marketplace or does it just prevent the other shipping settings apps from showing up? 
AFAIK they shouldn't have been showing up anyway because they would be disabled, but these changes seem to indicate differently.

Reply:
That's a good question. They only show up when you get the "No shipping methods configure. Configure now" with the link. Ideally this should only show up when you actually have the permissions to modify the shipping settings which is currently if you are a shop admin correct? But it's only currently checking for `["owner", "admin"]);`

I wonder if we can't make this only show up when you actually have permissions to configure shipping? i.e. are Marketplace Admin?

Reply:
For now, I can just back out that change, and we can deal with that separately?

Reply:
Yeah, I'm testing, I'll test with and without that, and approve one way or the other